packages/hardhat/test/e2e/periphery/controller-helper.ts

```diff
@@ -1356,4 +1460,635 @@ it("Close vault LP and open new one-siced LP with just oSQTH ", async () => {
+      // Get new vault and LP info
+      const vaultAfter = await controller.vaults(vaultId);
+      const tokenIdAfter = vaultAfter.NftCollateralId;
+      const positionAfter = await (positionManager as INonfungiblePositionManager).positions(tokenIdAfter)
+      const [amount0After, amount1After] = await (positionManager as INonfungiblePositionManager).connect(controllerSigner).callStatic.decreaseLiquidity({
+        tokenId: tokenIdAfter,
+        liquidity: positionAfter.liquidity,
+        amount0Min: 0,
+        amount1Min: 0,
+        deadline: Math.floor(await getNow(ethers.provider) + 8640000),
+      })
+      const wPowerPerpAmountInLPAfter = (isWethToken0) ? amount1After : amount0After;
+      const wethAmountInLPAfter = (isWethToken0) ? amount0After : amount1After;
+      // Changes
+      const depositorEthDiff = depositorEthBalanceAfter.sub(depositorEthBalanceBefore)
+      const depositorSqueethDiff = depositorSqueethBalanceAfter.sub(depositorSqueethBalanceBefore)
+      const vaultEthDiff = vaultAfter.collateralAmount.sub(vaultBefore.collateralAmount)
+      const vaultSqueethDiff = vaultAfter.shortAmount.sub(vaultBefore.shortAmount)
+      const lpEthDiff = wethAmountInLPAfter.sub(wethAmountInLPBefore)
+      const lpSqueethDiff = wPowerPerpAmountInLPAfter.sub(wPowerPerpAmountInLPBefore)
+      // Assertions
+
+      console.log('positionAfter.TickLower', positionAfter.tickLower)
+      console.log('positionAfter.TickUpper', positionAfter.tickUpper)
+      console.log('wPowerPerpAmountInLPBefore', wPowerPerpAmountInLPBefore.toString())
+      console.log('wethAmountInLPBefore', wethAmountInLPBefore.toString())
+      console.log('wPowerPerpAmountInLPAfter', wPowerPerpAmountInLPAfter.toString())
+      console.log('wethAmountInLPAfter', wethAmountInLPAfter.toString())
+      console.log('vaultSqueethDiff',vaultSqueethDiff.toString() )
+      console.log('vaultEthDiff',vaultEthDiff.toString() )
+      console.log('lpSqueethDiff', lpSqueethDiff.toString())
+      console.log('lpEthDiff', lpEthDiff.toString())
+      console.log('depositorSqueethDiff', depositorSqueethDiff.toString())
+      console.log('vaultBefore.collateralAmount', vaultBefore.collateralAmount.toString())
+      console.log('vaultBefore.shortAmount', vaultBefore.shortAmount.toString())
+      console.log('vaultAfter.collateralAmount', vaultAfter.collateralAmount.toString())
+      console.log('vaultAfter.shortAmount', vaultAfter.shortAmount.toString())
+      console.log('flashloanAmount',flashLoanAmount.toString())
+      console.log('wPowerPerpAmountInLPBefore', wPowerPerpAmountInLPBefore.toString())
+      console.log('depositorEthDiff', depositorEthDiff.toString())
+      console.log('depositorSqueethDiff', depositorSqueethDiff.toString())
+      console.log('wPowerPerpToSwap', wPowerPerpToSwap.toString())
+      
+      // Swap amount removed from LP
+      expect(vaultSqueethDiff.sub(wPowerPerpToSwap).eq(0)).to.be.true
```

Comment:
These 2 things aren't so related to eachother - this means that the minted amount is also swapped, but thats just by construction from the params - not sure they are really related.

Reply:
just checking that correct amount was minted (since we mint wPowerPerpToSwap) but I see your point. Will remove.

Reply:
maybe just shows that the correct amount was minted (the mint could have been wrong). I think the comment was misleading and changed